wpiunits/BUILD.bazel

```
@@ -0,0 +1,17 @@
+load("@rules_java//java:defs.bzl", "java_binary", "java_library")
+
+java_library(
+    name = "wpiunits-java",
```

nit: I think we expect wpiunits to always be Java-only, and the C++ units library to continue to live in wpimath. For convenience:
```suggestion
    name = "wpiunits",
```